acquisitions/src/main/resources/thunderjet/mod-orders/features/check-order-total-fields-calculated-correctly.feature

```diff
@@ -0,0 +1,148 @@
+Feature: Check that order total fields are calculated correctly
+
+  Background:
+    * url baseUrl
+    * callonce loginAdmin testAdmin
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json'  }
+    * callonce loginRegularUser testUser
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json'  }
+    * configure headers = headersAdmin
+
+    * callonce variables
+
+    ### Before All ###
+    * def previousFiscalYear = callonce uuid1
+    * def currentFiscalYear = callonce uuid2
+    * def codePrefix = callonce random_string
+    * def toYear = callonce getCurrentYear
+    * def fromYear = parseInt(toYear) - 1
+    * table fiscalYearDetails
+      | id                 | code                  | periodStart                   | periodEnd                     | series |
+      | previousFiscalYear | codePrefix + fromYear | fromYear + '-01-01T00:00:00Z' | fromYear + '-12-30T23:59:59Z' | 1      |
+      | currentFiscalYear  | codePrefix + toYear   | toYear + '-01-01T00:00:00Z'   | toYear + '-12-30T23:59:59Z'   | 2      |
+    * def v = callonce createFiscalYear fiscalYearDetails
+
+    * def ledgerId = callonce uuid3
+    * table ledgerDetails
+      | id       | fiscalYearId       | restrictEncumbrance | restrictExpenditures |
+      | ledgerId | previousFiscalYear | false               | false                |
+    * def v = callonce createLedger ledgerDetails
+
+    * def fundId = callonce uuid4
+    * table fundDetails
+      | id     | code     | ledgerId |
+      | fundId | "FUND-1" | ledgerId |
+    * def v = callonce createFund fundDetails
+
+    * table budgetDetails
+      | fundId | fiscalYearId       | allocated |
+      | fundId | previousFiscalYear | 9999999   |
+      | fundId | currentFiscalYear  | 9999999   |
+    * def v = callonce createBudget budgetDetails
+
+
+    ### Before Each ###
```

Comment:
it means it exetued in someway before each scenario?

Reply:
Divided callonce and call statements, just labeled Before All and Before Each to resemble unit tests and help understanding